release/xgboost_tests/oss_cluster_cpu_small.yaml

```diff
@@ -0,0 +1,46 @@
+cluster_name: ray-xgboost-release-cpu-small
+
+max_workers: 4
+
+upscaling_speed: 32
+
+idle_timeout_minutes: 15
+
+docker:
+    image: rayproject/ray:latest
+    container_name: ray_container
+    pull_before_run: true
+
+provider:
+    type: aws
+    region: us-west-2
+    availability_zone: us-west-2a
+    cache_stopped_nodes: false
+
+available_node_types:
+    cpu_4_ondemand:
+        node_config:
+            InstanceType: m5.xlarge
+        resources: {"CPU": 4}
+        min_workers: 3
+        max_workers: 3
+
+auth:
+    ssh_user: ubuntu
+
+head_node_type: cpu_4_ondemand
+worker_default_node_type: cpu_4_ondemand
+
+file_mounts: {
+    "~/xgboost_tests": "."
+}
+
+file_mounts_sync_continuously: false
+
+setup_commands:
+    - pip install pytest xgboost_ray
+    - sudo mkdir -p /data || true
+    - sudo chown ray:1000 /data || true
+    - rm -rf /data/classification.parquet || true
+    - cp -R /tmp/ray_tmp_mount/xgboost_tests ~/xgboost_tests || echo "Copy failed"
```

Comment:
consider moving all of this into some bash script (like `./setup-xgboost.sh`) and reusing it across tests